### PR TITLE
added disclaimer

### PR DESCRIPTION
**Pull request by eriktks**
Refs: #42 

## Description

Added legal disclaimer to README.md based on a [disclaimer found on Github](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository#disclaimer)

## To assess

This PR only involves added text. Reading the new paragraph in README.md is sufficient for evaluation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,4 +8,8 @@ GitHub Action that checks dependency licence issues using [ort](https://github.c
 
 ## Developer documentation
 
-See [README.dev.md](README.dev.md)
\ No newline at end of file
+See [README.dev.md](README.dev.md)
+
+## Disclaimer
+
+License Guard aims at providing insights into the license depencies of your package based on available information on open source licenses. We hope this information is helpful. However, we are not lawyers and we do make mistakes. Therefore License Guard provides information on an "as-is" basis and does not make warranties regarding this information. For any questions regarding the issues related to the licenses in your code, please consult a professional.
```
Comment (jspaaks) [inline README.md]:
```suggestion
## Disclaimer

`licenseguard` aims at providing insights into the license depencies of your package based on available information on open source licenses. We hope this information is helpful. However, we are not lawyers and we do make mistakes. Therefore `licenseguard` provides information on an "as-is" basis and does not make warranties regarding this information. For any questions regarding the issues related to the licenses in your code, please consult a professional.    
```

**Review by jspaaks — CHANGES_REQUESTED** [review]
Thanks for the PR. I left a small suggestion --otherwise good to go

**Comment by eriktks** [comment]
Ok, the suggested change was committed

**Review by jspaaks — APPROVED** [review]
perfect, :trophy:

**Pull request merged**